pkgs/applications/virtualization/imgcrypt/default.nix

```
@@ -0,0 +1,33 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+buildGoModule rec {
```

Better readability for humans.

```suggestion
{ buildGoModule, fetchFromGitHub, lib }:

buildGoModule rec {
```